ocs_ci/helpers/helpers.py

```diff
@@ -1805,13 +1805,14 @@ def change_default_storageclass(scname):
     ocp_obj = ocp.OCP(kind="StorageClass")
     if default_sc:
         # Change the existing default Storageclass annotation to false
-        patch = (
-            ' \'{"metadata": {"annotations":'
-            '{"storageclass.kubernetes.io/is-default-class"'
-            ':"false"}}}\' '
-        )
-        patch_cmd = f"patch storageclass {default_sc} -p" + patch
-        ocp_obj.exec_oc_cmd(command=patch_cmd)
+        for sc in default_sc:
```

Comment:
What if `default_sc` is not a list?

Reply:
`get_default_storage_class()` in line 1770 returns a list of SCs and is assigned to `default_sc`. In case there are no default SCs in the cluster, this code will not be executed and it will only run the patch command to change the new SC as the default SC.